ui/src/app/cohort-review/participant-status/participant-status.component.ts

```diff
@@ -69,7 +79,9 @@ export class ParticipantStatusComponent implements OnInit, OnDestroy {
   }
 
   private callApi = ([status, pid]): Observable<ParticipantCohortStatus> => {
-    this.changingStatus = true;
+    if (status === '') {
+      return Observable.of(this.participant);
```

Comment:
This breaks contract with the function signature (it returns `Observable<Participant>`).  I like the goal though (which I presume is to short-circuit the api call and not make it if the value isn't valid).  I suggest moving this sentiment to a `.filter` on the source Observable.  E.g. change

```
const statusChanger = this.statusControl.valueChanges
  .filter(value => value !== '')
  .withLatestFrom(participantId)
```

We could even change that predicate to check that the value is valid, rather than that it isn't empty.

Reply:
Nice... i didn't think of the filter. Done.